koku/reporting/provider/ocp/models.py

```diff
@@ -268,6 +268,125 @@ class Meta:
     node_role = models.TextField(null=True)
 
 
+class OCPNetworkSummaryP(models.Model):
+    """Store summary of bytes in, bytes out, and costs per cluster per day"""
+
+    class PartitionInfo:
+        partition_type = "RANGE"
+        partition_cols = ["usage_start"]
+
+    class Meta:
+        db_table = "reporting_ocp_network_summary_p"
+        indexes = [models.Index(fields=["usage_start"], name="ocp_net_summ_usage_start")]
+
+    id = models.UUIDField(primary_key=True)
+
+    cluster_alias = models.TextField(null=True)
+    cluster_id = models.TextField()
+    # Cost model fields need to exist even though they are not relevant to this model
+    cost_model_cpu_cost = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    cost_model_memory_cost = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    cost_model_rate_type = models.TextField(null=True)
+    cost_model_volume_cost = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    data_source = models.CharField(max_length=64, null=True)
+    distributed_cost = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    infrastructure_data_in = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    infrastructure_data_out = models.DecimalField(max_digits=33, decimal_places=15, null=True)
```

Comment:
These models should be specific aggregations/subsets of the `OCPUsageLineItemDailySummary` table, we can't add fields here that don't exist on the base table because we wont have the data to populate from.

Also when a query for the future API endpoint cannot be resolved by the summary tables it will fall back to the base table (assuming we match the behavior of all the other APIs), so if a field is present here but not there, we can't fall back appropriately.

Reply:
That's an unfortunate relationship that isn't very apparent. It really seems like these fields make sense on this model. I guess we need to add them to `OCPUsageLineItemDailySummary`.